src/book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+require_relative './rental'
+
+class Book
+  attr_accessor :date, :title, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = nil
```

Comment:
- [ ] Kindly notice that it is better to rename it to `rentals` instead of `rental` as you are going to add many rentals to it.
- [ ] Also notice that it is better to make the `rentals` equal to empty array `[]` so it makes it easy to push data in it.

```ruby
@rentals = []
```